src/Components/Coins/CoinsButton.js

```diff
@@ -20,7 +20,7 @@ export default function CoinsButton() {
 
   return (
     <div id="coinsButtonContainer">
-      <GeneralButton id="coinsButton" uppercased outlined onButtonClick={() => onCoinsButtonClick(isCoinsTabOpen)}>
+      <GeneralButton id="coinsButton" uppercased outlined onButtonClick={() => onCoinsButtonClick()}>
```

Comment:
good catch
but now - it's an anonymous func. that operates another func. that receives not parameters,
it'll still work, and even be more concise if it would've been
`onButtonClick={onCoinsButtonClick}` instead of 
`onButtonClick={() => onCoinsButtonClick()}`.
can you please make that change?